packages/core/services/DatabaseService/index.ts

```diff
@@ -529,4 +529,23 @@ export default abstract class DatabaseService {
             throw err;
         }
     }
+
+    public async addNewColumn(datasourceName: string, columnName: string, description?: string) {
+        try {
+            await this.execute(
+                `ALTER TABLE "${datasourceName}" ADD COLUMN "${columnName}" VARCHAR;`
+            );
+
+            // Cache is now invalid since we added a column
+            this.dataSourceToAnnotationsMap.delete(datasourceName);
+
+            if (description?.trim() && this.existingDataSources.has(this.SOURCE_METADATA_TABLE)) {
+                await this
+                    .execute(`INSERT INTO "${this.SOURCE_METADATA_TABLE}" ("Column Name", "Description")
+                        VALUES ('${columnName}', '${description}');`);
+            }
+        } catch (err) {
+            throw err;
```

Comment:
lol catch to throw it? I think we can just remove this then

Reply:
Good catch lol I think this is a debugging artifact